README.md

```diff
@@ -230,7 +229,7 @@ This is typically used in conjunction with -b.
 
 --use-jep319-certs
```

Comment:
I think this option has been fully deprecated, but we can deal with that under a separate PR

Reply:
[It looks like we use this a lot](https://github.com/AdoptOpenJDK/openjdk-build/search?q=--use-jep319-certs&type=Code). Does it not have any effect on the builds anymore?

Reply:
Moving discussion to separate issue https://github.com/AdoptOpenJDK/openjdk-build/issues/2180 - the change was backed out and apparently hasn't been redone yet.